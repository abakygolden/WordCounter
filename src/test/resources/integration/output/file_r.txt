R 8
RASKE 1
RAZ 1
RE 3
RECUERDEN 1
REGERINGER 1
REN 1
RESPETEN 1
RET 1
RIGOR 1
RING 1
RIVER 2
ROBA 1
ROBO 1
ROCK 2