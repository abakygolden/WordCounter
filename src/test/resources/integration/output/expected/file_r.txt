RAMBUTAN 4
RAMEN 4
RASPBERRY 4
RATATOUILLE 4
RIBS 4
RICE 8
RISOTTO 4
ROAST 4
ROLL 4
ROLLS 8